website/src/pages/extensions/index.tsx

```
@@ -1,24 +1,682 @@
+      </div>
+
+      <div className="container mx-auto px-5 py-8">
+        <div className="flex flex-col lg:flex-row gap-8">
+          {/* Sidebar */}
+          <div className="w-full lg:w-64 flex-shrink-0">
+            <div className="bg-white dark:bg-charcoal-800 rounded-lg lg:sticky lg:top-8">
+              <div className="flex items-center justify-between mb-4">
+                <h2 className="text-lg font-semibold text-charcoal-300 dark:text-white mb-0">Filters</h2>
+                <button
+                  onClick={() => setShowFilters(!showFilters)}
+                  className="lg:hidden bg-gray-100 dark:bg-charcoal-700 hover:bg-gray-200 dark:hover:bg-charcoal-600 text-gray-600 dark:text-gray-300 px-3 py-2 rounded-lg transition-colors duration-200 flex items-center justify-center">
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)